runelite-client/src/main/java/net/runelite/client/plugins/cooking/CookingPlugin.java

```diff
@@ -121,6 +126,11 @@ public void onGameTick(GameTick gameTick)
 			{
 				fermentTimerSession = null;
 			}
+
+			if (client.getLocalPlayer().getAnimation() == COOKING_WINE)
```

Comment:
You should never need to check for this _and_ check the difference in time since the last wine making action. Place this on L125 and change the previous `if` to `else if`.

Reply:
Fixed.